<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smu1_idc</class>
 <widget class="QDialog" name="smu1_idc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>170</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>260</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>details</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:StopM7</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:StopM8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>490</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>more</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>30</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Stop All</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>330</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>details</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>70</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>relative</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>100</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>details</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:StopM1</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>430</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:StopM9</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>absolute</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:StopM6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Motor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>IDc</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>SMU41_1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>420</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>details</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:LsnM6</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:LspM6</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:LsnM7</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:LspM7</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:LsnM8</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:LspM8</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:LsnM9</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:LspM9</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m1Axis_2" native="true">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string/>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdRelM1</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:RelM1</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m6Axis_2" native="true">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>170</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string/>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdRelM6</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:RelM6</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m7Axis_2" native="true">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>250</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string/>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdRelM7</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:RelM7</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m8Axis_2" native="true">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>330</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string/>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdRelM8</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:RelM8</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m9Axis_2" native="true">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>420</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string/>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdRelM9</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:RelM9</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m1Axis_1" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>M1 Switch</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdPosM1</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:AbsM1</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m6Axis_1" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>M6 Left Rz</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdPosM6</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:AbsM6</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m7Axis_1" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>M7 Left Rz</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdPosM7</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:AbsM7</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m8Axis_1" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>M8 Left Rz</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdPosM8</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:AbsM8</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m9Axis_1" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>M9 Left Rz</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdPosM9</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:AbsM9</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCALabel</class>
   <extends>QLabel</extends>
   <header>qcalabel.h</header>
  </customwidget>
  <customwidget>
   <class>QCAPushButton</class>
   <extends>QPushButton</extends>
   <header>qcapushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QCAAxis</class>
   <extends>QWidget</extends>
   <header>qcaaxis.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
